repo.json

```diff
@@ -2415,6 +2415,11 @@
           "version": "1.1.0",
           "commit": "0d2ee17f7dd7516affec5b1eca502d1bc1e2d212",
           "url": "https://github.com/digrich/bubblechart-panel"
+        },
+        {
+          "version": "1.2.0",
```

Comment:
The version in the [plugin.json file](https://github.com/digrich/bubblechart-panel/blob/e86d545dcbb3cf63b4233ffb056de58e7f102c4d/dist/plugin.json#L25) needs to be updated to 1.2.0,

Reply:
Updated.